### DISCUSSION TITLE
how can I disable ADMIN GUI in kong.conf

### DISCUSSION THREAD

**Discussion by shilaidun** [Kong/kong#12252] [Help] [answered]
how can I disable ADMIN GUI in kong.conf 
in other words，I do not want to enable 8002 or 8445 listening

**Comment by ms2008** [comment, marked as answer]
Just set `admin_gui_listen` to off, or you can configure it through environment variables, like this: `export KONG_ADMIN_GUI_LISTEN=off`.
